test/cookbooks/osl-nginx-test/recipes/default.rb

```diff
@@ -1,5 +1,9 @@
 include_recipe 'osl-nginx'
 
+file '/etc/nginx/conf.d/test-kitchen.conf' do
+  content 'server_names_hash_bucket_size 128;'
```

Comment:
This needs to notify restarting the nginx service so the config is active. Can you please add that?